named.go

```diff
@@ -224,29 +224,47 @@ func bindStruct(bindType int, query string, arg interface{}, m *reflectx.Mapper)
 	return bound, arglist, nil
 }
 
-var valueBracketReg = regexp.MustCompile(`(?i)VALUES\s*(\([^(]*.[^(]\))`)
+var valuesReg = regexp.MustCompile(`\)\s*(?i)VALUES\s*\(`)
```

Comment:
Should we also care for inserts without an explicit column list here, e.g. `insert into tblname values (...)`?

Reply:
Oh. Thanks for pointing out. But I dont find a good way to both satisfying that case and the case when the name of table containing suffix: "values".

Reply:
I think `\W+(?i)VALUES\s*\(` should handle most cases, including:
```
INSERT INTO foo_values (a,b,c,d) VALUES (?, ?, ?, ?)
INSERT INTO foo_values VALUES (?, ?, ?, ?)
INSERT INTO `values`VALUES (?, ?, ?, ?)
```
While generally being a bad idea, I have no doubt that somebody does implement it, given such a wide adoption of sqlx)